src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/transactionDataListItemStructure.api.ts

```diff
@@ -0,0 +1,71 @@
+import { type Hash } from 'viem';
+import { IconType, type AvatarIconVariant, type IDataListItemProps } from '../../../../../core';
+
+export enum TxStatusCode {
```

Comment:
To follow the naming of `TransactionType`, I would call this `TransactionStatus`
```suggestion
export enum TransactionStatus {
```